modin/experimental/cloud/rayscale.py

```diff
@@ -167,7 +167,7 @@ def _get_python_version():
         major = sys.version_info.major
         minor = sys.version_info.minor
         micro = sys.version_info.micro
-        return f"{major}.{minor}.{micro}"
+        return [f"python>{major}.{minor}", f"python<={major}.{minor}.{micro}"]
```

Comment:
```suggestion
        return [f"python>={major}.{minor}", f"python<={major}.{minor}.{micro}"]
```
otherwise might break on `x.y.0` version

Another option is requiring just `{major}.{minor}`, omitting `{micro}` altogether

Reply:
If I specify the python version as `python==3.7`, then the conda tries to install the minimum available version- `3.7.0`. So just commit your suggestions.